webapp/app/model/components/__init__.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List, Optional, Union
+
+import humps
+from pydantic import BaseModel, Extra
+
+
+class ComponentProps(BaseModel, extra=Extra.forbid):
+    def camelized_dict(self):
+        return humps.camelize(self.dict())
+
+
+class StepHeaderProps(ComponentProps):
```

Comment:
Why did you put this into the __init__ and not into something like `component_props`?

Reply:
What would be the benefit of doing it like that?

Reply:
I'd knew what to expect from the file? :D